com.github.k4zmu2a.spacecadetpinball.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
there is a desktop file upstream, so why including one here?

Reply:
I feel the upstream is less close to the original. Maybe can we do something about this on upstream @k4zmu2a?

Reply:
I think it's better to use the upstream one, and open a pr to improve this file at upstream repo

Reply:
https://github.com/k4zmu2a/SpaceCadetPinball/pull/111

Reply:
Merged